### PR TITLE
Rewrite.

### PR DESCRIPTION
**Pull request by ben-eb**
Supersedes https://github.com/MoOx/reduce-css-calc/pull/21.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import valueParser from 'postcss-value-parser';
-import { parser } from './parser';
+
+import { parser } from './parser'; // eslint-disable-line
```
Comment (ben-eb) [inline src/index.js]: This is due to `parser` only being available in the `dist` folder, it's automatically generated.

**Comment by ben-eb** [comment]
@MoOx Also would you like me to remove the semicolons from this PR? Would like to get this released ASAP so we can update postcss-calc to PostCSS 6.

**Comment by MoOx** [comment]
Do whatever you want :)

**Comment by ben-eb** [comment]
Look good for a 2.0.0 release? 😃

**Comment by MoOx** [comment]
I don't have time to review. I will trust you on this. Be my guest.

**Pull request merged**